zh-CN.json

```diff
@@ -0,0 +1,284 @@
+    "commands": "命令",
+    "notifications": "直播通知",
+    "counters": "关注者计数器",
+    "subLeaderboard": "订阅者排行榜",
+    "subRole": "订阅者角色",
+    "misc": "杂项",
+    "madeBy": "由[Sean Sattler](<https://sattler.dev> \"打开一个新标签页sattler.dev\")制作",
+    "notAffiliated": "*与<:Kick_Green:1118592781930221648> [Kick](<https://kick.com> \"打开一个新标签页kick.com\")无关*"
+  },
+  "insightsCommand": {
+    "commandData": {
+      "description": "查看机器人的洞察！"
+    },
+    "servers": "服务器",
+    "version": "版本",
+    "streamers": "主播",
+    "ramUsage": "RAM使用",
+    "liveNotifications": "设置直播通知",
+    "liveLeaderboards": "设置订阅者排行榜\n",
+    "leaderboards": "排行榜",
+    "setupCounters": "设置计数器",
+    "counters": "计数器",
+    "onlineSince": "在线自"
+  },
+  "inviteCommand": {
+    "commandData": {
+      "description": "想要邀请机器人到你的服务器？使用这个命令！"
+    },
+    "inviteText": "<:PlusIcon:1118592778897719347> **[点击这里邀请机器人到你的服务器！](https://discord.com/api/oauth2/authorize?client_id=1118578278299611246&permissions=275146828816&redirect_uri=https%3A%2F%2Fkick.bot%3Futm_source%3Ddiscord-app-invite&response_type=code&scope=bot%20applications.commands%20identify \"打开一个新标签页邀请链接！\")**"
+  },
+  "leaderboardCommand": {
+    "commandData": {
+      "description": "为主播添加/移除或更新一个排行榜。",
+      "optionStreamerRemove": "你想要移除的主播的用户名。",
+      "listDescription": "列出这个服务器上所有的排行榜。",
+      "addDescription": "为主播添加一个排行榜。",
+      "removeDescription": "移除一个主播的排行榜。",
+      "editDescription": "编辑一个主播的排行榜。",
+      "optionChannel": "排行榜的频道。",
+      "optionStyle": "排行榜的样式。",
+      "style3": "前3名排行榜样式。",
+      "style5": "前5名排行榜样式。",
+      "style10": "前10名排行榜样式。"
+    },
+    "alreadyLeaderboard": "这个主播已经有一个排行榜了！",
+    "sendError": "尝试发送排行榜时我遇到了一个错误！\n*错误：{error}*",
+    "added": "<#{channel}>现在是**{streamer}**的排行榜频道了！",
+    "channelOrStyle": "你必须提供一个频道或样式来编辑！",
+    "noLeaderboard": "这个主播没有排行榜！",
+    "edited": "成功编辑了<#{channel}>的配置！",
+    "removed": "已移除**{streamer}**的排行榜！\n*是由<@{addedById}>添加的。*",
+    "noLeaderboards": "这个服务器上没有排行榜！"
+  },
+  "linkCommand": {
+    "commandData": {
+      "description": "将你的Kick.com账户与你的Discord账户链接！"
+    },
+    "alreadyLinked": "你已经与一个Kick.com账户链接了！使用下面的按钮重新链接它。"
+  },
+  "listCommand": {
+    "commandData": {
+      "description": "查看这个服务器上由机器人跟踪的所有主播！"
+    },
+    "emptyList": "这个服务器上没有设置的主播！"
+  },
+  "removeCommand": {
+    "commandData": {
+      "description": "从机器人中移除一个主播！",
+      "optionStreamer": "你想要移除的主播的用户名。"
+    },
+    "removed": "**{streamer}**已经从机器人中移除了！\n*是由<@{addedById}>添加的。*"
+  },
+  "subscriberRoleCommand": {
+    "commandData": {
+      "description": "为主播添加一个订阅者角色。",
+      "optionStreamer": "订阅者应该能够获得角色的主播的用户名。",
+      "optionRole": "你想要给订阅者的角色。",
+      "optionMessage": "如果你想要有一个自定义消息。"
+    },
+    "embed": {
+      "title": "订阅者角色",
+      "description": "要获得**{streamer}**的订阅者角色，你只需要点击下面名为**Sub Role**的按钮。"
+    },
+    "added": "我已经将订阅者角色嵌入发送到了<#{channel}>！\n*信息：如果你想要删除设置，只需删除消息和角色，机器人将自动从数据库中删除设置。*",
```

Comment:
just noticed also for every placeholder or like <#id> too this all needs spaces else Discord does not render it